modules/ding_wayf_dk/includes/DingWayfGateway.class.inc

```diff
@@ -0,0 +1,118 @@
+<?php
+/**
+ * @file
+ * Simple class for communication with the SimpleSAMLphp DBC gateway.
```

Comment:
This module is able to communicate with wayf.dk directly or with the changes requested by DDB, the gateway at DBC that them communicates with DBC.

I'm not a fan of this extra layer of complexity, but if this should not be part WAYF module it would require a whole new module that has noting to do with wayf.dk.